lib/api/unittest/CAnomalyJobTest.cc

```diff
@@ -186,11 +186,10 @@ BOOST_AUTO_TEST_CASE(testBadTimes) {
         // Test with no time field
         model::CLimits limits;
         api::CAnomalyJobConfig jobConfig;
+        jobConfig.analysisConfig().addDetector("value", "", "", "", "greenhouse");
```

Comment:
```suggestion
        jobConfig.analysisConfig().addDetector("metric", "value", "", "", "greenhouse");
```

This was using implicit `metric` which we should move away from.